static/src/stylesheets/amp/_d2-comments.scss

```diff
@@ -0,0 +1,50 @@
+/* Comments
+   ========================================================================== */
+
+.d2-comment-embedded {
+    background-color: #ffffff;
+    font-family: sans-serif;
+    overflow: hidden;
+    position: relative;
+    margin-bottom: 1.5em;
+    padding-top: 15px;
+    font-size: 14px;
+    padding-left: 28px; // aligns with top level comment's body
+    border-left: 8px solid $neutral-5;
+    clear: left;
+}
+
+.d2-avatar-image {
+    float: left;
+    margin-right: 15px;
+    margin-bottom: 12px;
+
+    @include mq(tablet) {
+        width: 60px;
+        height: 60px;
+    }
+}
+
+.d2-right-col {
+    position: relative;
+    top: -3px;
+    padding-right: 16px; // prevent colliding with badge
+}
+
+.d2-username {
+    font-family: $f-serif-text;
+}
+
+.d2-permalink .d2-datetime {
+    color: $neutral-2;
+    font-size: .8em;
+    font-family: $f-sans-serif-text; // Force font when used outside of comment context
+}
+
+.d2-permalink .d2-datetime:hover {
+    border-bottom: 1px dotted #757575;
```

Comment:
I pulled this down and ran it so I could see the hover states, it doesn't look great to me because the lines don't seem to match up. This is what it looks like:

![image](https://cloud.githubusercontent.com/assets/8774970/22212053/266ceaf4-e187-11e6-98bd-f4fcbbc3172a.png)

I would opt for removing it anyway because AMP is only really for mobile (no hover)